Description: Build with no default features enabled
Author: Mike Hommey <glandium@debian.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: git-cinnabar/Cargo.toml
===================================================================
--- git-cinnabar.orig/Cargo.toml
+++ git-cinnabar/Cargo.toml
@@ -46,8 +46,6 @@ repository = "https://github.com/glandiu
 [features]
 compile_commands = []
 default = [
-    "full-version",
-    "version-check",
 ]
 full-version = [
     "dep:concat_const",
